lib/airbrake/rake/tasks.rb

```diff
@@ -106,8 +106,19 @@
            " environment will be used."
     end
 
+    unless (repo = ENV['REPOSITORY_URL'])
+      repo = `git remote get-url origin 2>/dev/null`.chomp
+      if repo.empty?
+        puts "Airbrake couldn't identify your app's repository."
```

Comment:
I was thinking it should be more generic because it means two checks have failed, they didn't set the `REPOSITORY_URL` env var _and_ we couldn't get the url for the `origin` remote.